pkg/reconciler/pipelinerun/pipelinerun.go

```diff
@@ -385,9 +387,12 @@ func (c *Reconciler) resolvePipelineState(
 					"Pipeline %s/%s can't be Run; it contains Tasks that don't exist: %s",
 					pipelineMeta.Namespace, pipelineMeta.Name, nfErr)
 			} else {
```

Comment:
[Severity-low]: It might be helpful to include the name of the PipelineTask that failed to resolve in the error message. This would make it easier to identify the source of the problem.
```suggestion
 err = pipelineErrors.WrapUserError(fmt.Errorf("PipelineTask %q: %w", pt.Name, err))
```